lib/hanami/config.rb

```diff
@@ -127,6 +127,45 @@ class Config
     #   @since 2.0.0
     setting :base_url, default: "http://0.0.0.0:2300", constructor: ->(url) { URI(url) }
 
+    # @!attribute [rw] render_errors
+    #   Sets whether to catch exceptions and render error pages.
+    #
+    #   For HTML responses, these error pages are in `public/{404,500}.html`.
+    #
+    #   Defaults to `true` in production mode, `false` in all others.
+    #
+    #   @return [Boolean]
+    #
+    #   @api public
+    #   @since 2.1.0
+    setting :render_errors, default: false
```

Comment:
These settings will be unneeded/unused when using Hanami in "non-web" mode (i.e. with hanami-router and hanami-controller removed from the app's `Gemfile`), but I decided to put them at the top-level here (rather than e.g. nest them inside `config.router` or some other namespace) because they're kind of a cross-cutting concern and I don't think we need to go nuts creating namespaces for general settings just yet.

In addition, they're not harming anyone if they exist here unused in the case of non-web apps.

For future major versions of Hanami we might consider a different approach, but this feels appropriate to me for now.